src/cloud-api-adaptor/podvm/qcow2/copy-files.sh

```diff
@@ -30,6 +30,7 @@ sudo cp -a /tmp/files/pause_bundle /
 # Copy the kata-agent OPA policy files
 sudo mkdir -p /etc/kata-opa
 sudo cp -a /tmp/files/etc/kata-opa/* /etc/kata-opa/
+sudo ln -s /run/peerpod/policy.rego /etc/kata-opa/default-policy.rego
```

Comment:
The default-policy.rego is deleted in this PR. Is this intended?

Reply:
Yes, that's the point :-)
The problem is that it is a symbolic link, pointing outside of the repo (/run/...), so our build system complains that it breaks our policy.
To fix the build, I'm removing the link, BUT I'm trying to restore it during the copy file operation.

Reply:
@littlejawa I'm just concerned this might conflict with some upstream changes in the future. Lets say upstream changes the symlink to something else, we will catch this?

With CPaaS this error was fixed by setting a cachito flag to ignore the warning about symlink outside the repo. I know konflux uses cachi2 instead... have you asked konflux team about this?

Reply:
Revising the flag, yes, you are right. My concern is valid with cpaas as well. We just need to pay attention to this file.